edx/edx-bulk-grades
# add course role filtering to grade export

**Description:** Add an excludedCourseRoles parameter to the bulk grades download api call to match lms gradebookView endpoint and gradebook mfe behavior

**JIRA:** [EDUCATOR-5571](https://openedx.atlassian.net/browse/EDUCATOR-5571)

**Testing instructions:** 
 - Install bulk grades into local lms.
 - Choose a course with some users configured as having course roles.
 - Make a request to  `LMS URL`/api/bulk_grades/course/`course_id`/?excludedCourseRoles=`role`
 - (Staff = staff, Admin = instructor, Beta = beta_testers)
 - Verify that users with the given role(s) are excluded from the response, and that all users with roles are ignored if "all" is specified.
 
**Merge checklist:**

**Post merge:**
      finished.
